### PR TITLE
Replace ciris with zio-config

### PR DESCRIPTION
**Pull request by amir**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -29,6 +28,7 @@ val V = new {
   val `zio-kafka`        = "2.4.2"
   val `zio-oci-os`       = "0.6.0"
   val `zio-s3`           = "0.4.2.4"
+  val `zio-config`       = "3.0.7"
```
Comment (sirocchj) [inline build.sbt:31]: mind put them in order? :)

File: build.sbt [hunk 2]
```diff
@@ -110,10 +110,10 @@ lazy val core = project
       "com.fasterxml.jackson.core"             % "jackson-databind"             % V.`jackson-databind`,
       "dev.zio"                               %% "zio-interop-cats"             % V.`zio-interop`,
       "dev.zio"                               %% "zio-kafka"                    % V.`zio-kafka`,
+      "dev.zio"                               %% "zio-config"                   % V.`zio-config`,
```
Comment (sirocchj) [inline build.sbt:113]: same

**Review by sirocchj — APPROVED** [review]

**Comment by amir** [comment]
I'll investigate whether we can use https://zio.dev/reference/configuration/

**Comment by amir** [comment]
alternative approach: https://github.com/laserdisc-io/tamer/pull/1301

**Review by sirocchj — APPROVED** [review]

**Review by sirocchj — APPROVED** [review]

**Comment by sirocchj** [comment]
@amir whoch one do you want to go for? #1301 or this one?

**Comment by amir** [comment]
@sirocchj going with the other one

**Pull request closed without merging**